starknet_py/tests/e2e/conftest.py

```diff
@@ -469,6 +471,113 @@ async def cairo_serializer(gateway_account_client: AccountClient) -> CairoSerial
     return CairoSerializer(identifier_manager=contract.data.identifier_manager)
 
 
+@pytest_asyncio.fixture(scope="module")
+async def deployer_address(gateway_client: AccountClient) -> int:
```

Comment:
Do we still need this? https://github.com/Shard-Labs/starknet-devnet/pull/324

Reply:
Only for a while. They are deploying it to the wrong address https://github.com/Shard-Labs/starknet-devnet/issues/326

Reply:
Hmmmm do you know if they plan to change that?

Reply:
I hope so, but I don't understand the last message from the linked issue 😅

Reply:
I think we might want to wait for devnet to avoid confusion 🤔 Please contact them to see what's the plan.